client/src/pages/ScanStore.tsx

```diff
@@ -0,0 +1,137 @@
+import React, {useEffect, useState} from 'react'
+import StoreHeader from './../components/StoreHeader'
+import Divider from '@material-ui/core/Divider'
+
+interface Item {
```

Comment:
nit: Consider abstracting out if these type definitions may be reused in other components (same for rest)

Reply:
refactored interface definitions and constructors into `interfaces`